Documentation/guides/messages/xa1037.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Xamarin.Android error XA1037
+description: XA1037 error code
+ms.date: 5/17/2023
+---
+# Xamarin.Android error XA1037
+
+## Example messages
+
+```
+error XA1037: AndroidManifest.xml //uses-sdk/@android:minSdkVersion '29' does not match the $(SupportedOSPlatformVersion) value '21' in the project file (if there is no $(SupportedOSPlatformVersion) value in the project file, then a default value has been assumed). Either change the value in the AndroidManifest.xml to match the $(SupportedOSPlatformVersion) value, or remove the value in the AndroidManifest.xml (and add a $(SupportedOSPlatformVersion) value to the project file if it doesn't already exist).
```

Comment:
This is a *long* error message, and doesn't render "nicely" on GitHub (*lots* of horizontal scrolling).  For "doc" purposes we should split this up "reasonably", e.g. every sentence on a newline.

Then there's the IDE side: does this line wrap "reasonably" within the IDE Errors panel?  If not, we'll want to make this a multi-line error message, a'la XA0031: https://github.com/xamarin/xamarin-android/blob/c701991292cdd0471e2affcf0fac7d8743def7a5/src/Xamarin.Android.Build.Tasks/Properties/Resources.resx#L183-L186

Reply:
Updated this to put each sentence on a new line.